source/manage/conf-postgres.rst

```diff
@@ -77,6 +77,11 @@ priority. Here is the previous example modified to use these flags::
 
      pmm-admin add postgresql --username=pmm --password=pmm --service-name=postgres --host=127.0.0.1 --port=270175432
```

Comment:
I guess it's better to use real port 5432, as 270175432 is bigger than Unix allow (2^16)